contracts/ArchetypeLogic.sol

```diff
@@ -67,24 +67,28 @@ struct Discount {
 struct Config {
   string baseUri;
   address affiliateSigner;
-  address ownerAltPayout; // optional alternative address for owner withdrawals.
-  address superAffiliatePayout; // optional super affiliate address, will receive half of platform fee if set.
   uint32 maxSupply;
   uint32 maxBatchSize;
   uint16 affiliateFee; //BPS
-  uint16 platformFee; //BPS
   uint16 defaultRoyalty; //BPS
   Discount discounts;
 }
 
+struct PayoutConfig {
+  uint16 ownerBps;
+  uint16 platformBps;
+  uint16 partnerBps;
```

Comment:
can you add brief comments here to explain what each share represents? will help clarify the intention